config/services.php

```diff
@@ -66,24 +81,24 @@
             ->alias(StoryblokClientInterface::class, StoryblokClient::class)
 
         ->set(DatasourceEntriesApi::class)
-        ->alias(DatasourceEntriesApiInterface::class, DatasourceEntriesApi::class)
+            ->alias(DatasourceEntriesApiInterface::class, DatasourceEntriesApi::class)
```

Comment:
it would be nice to have these indention changes in a separate PR to keep this one smaller